src/containers/Preview/ProjectPreview/index.js

```diff
@@ -17,6 +18,8 @@ import ActivityCard from 'components/ActivityCard';
 
 import './style.scss';
 
+const history = createBrowserHistory();
```

Comment:
Please do not create history here, it will create new history and generate issue
Use withRouter to get history as props
And it would be better to create history on store and import it from other files

Reply:
changed